app/forms/hyrax/audio_form.rb

```diff
@@ -19,5 +19,15 @@ def self.model_attributes(_)
     def title
       super.first || ""
     end
+
+    def representative_id
+      return if media_files.first[1].nil?
```

Comment:
doesn't this just return `media_files.first[1]` no matter what?

I.e. it's equivalent to

```ruby
  return nil if media_files.first[1].nil?
  media_files.first[1]
```

and therefore

```ruby
  media_files.first[1]
```

Also, I'm wondering whether we really want to hard code these, or just set them. Other models do `work.representative = some_file` and `work.thumbnail = some_file`, e.g. at https://github.com/samvera/hyrax/blob/master/app/actors/hyrax/actors/file_set_actor.rb#L77-L78

It seems like a good idea to continue that pattern. Hard coding these will lead to unexpected results if we try to set this elsewhere.